ajax/libs/bootstrap-vue/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "bootstrap-vue",
+  "filename": "bootstrap-vue.common.min.js",
+  "version": "1.3.0",
+  "description": "Quickly integrate Bootstrap 4 components with Vue.js",
+  "license": "MIT",
+  "homepage": "https://bootstrap-vue.github.io",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bootstrap-vue/bootstrap-vue.git"
+  },
+  "author": "pi0 <pooya@pi0.ir>",
```

Comment:
The "AUTHOR" pi0 is fine here because the refer link show https://github.com/bootstrap-vue/bootstrap-vue/graphs/contributors?from=2016-10-02&to=2016-10-04&type=c

Reply:
Hey. I think it is okay for the team keeping this if it's just for internal purposes. Anyway would be awesome if we can list all of the team :)